jersey-servers/src/main/java/com/palantir/remoting2/servers/jersey/JsonExceptionMapper.java

```diff
@@ -88,6 +90,14 @@ public final Response toResponse(T exception) {
             builder.type(MediaType.TEXT_PLAIN);
             builder.entity(exceptionMessage);
         }
+
+        if (exception instanceof NotAuthorizedException) {
```

Comment:
aside: of the subtypes of `ClientErrorException`, only `NotAuthorizedException` and `NotAllowedException` seem to require special handling (i.e. some response header). I'm not convinced we're probably tackling `NotAllowedException` (https://httpstatuses.com/405)